source/reference/minio-mc/mc-cat.rst

```diff
@@ -105,22 +105,84 @@ Parameters
       :start-after: start-version-id-desc
       :end-before: end-version-id-desc
 
-.. mc-cmd:: --encrypt-key
+.. mc-cmd:: --enc-kms
+
+   Encrypt or decrypt objects using server-side :ref:`SSE-KMS encryption <minio-sse>` with client-managed keys.
+   
+   The parameter accepts a key-value pair formatted as ``KEY=VALUE``
+
+   - ``KEY`` must contain the full path to the object as ``alias/bucket/path/object``.
+   - ``VALUE`` must contain the 32-byte Base64-encoded data key to use for encrypting object(s).
+
+   The ``VALUE`` must correspond to an existing data key on the external KMS.
+   See the :mc-cmd:`mc admin kms key create` reference for creating data keys.
+
+   For example:
+
+   .. code-block:: shell
+
+      --enc-kms "myminio/mybucket/prefix/object.obj=bXktc3NlLWMta2V5Cg=="
+
+   You can specify multiple encryption keys by repeating the parameter.
+
+   Specify the path to a prefix to apply encryption to all matching objects at that path:
+
+   .. code-block:: shell
+
+      --enc-kms "myminio/mybucket/prefix/=bXktc3NlLWMta2V5Cg=="
+
+.. mc-cmd:: --enc-s3
    :optional:
 
-   Encrypt or decrypt objects using server-side encryption with
-   client-specified keys. Specify key-value pairs as ``KEY=VALUE``.
+   Encrypt or decrypt objects using server-side :ref:`SSE-S3 encryption <minio-sse>` with KMS-managed keys.
+   Specify the full path to the object as ``alias/bucket/prefix/object``.
+
+   For example:
+
+   .. code-block:: shell
+
+      --enc-s3 "myminio/mybucket/prefix/object.obj"
```

Comment:
There's a newer PR that ensures this works as expected, no need to mention it afaik.